setup.py

```diff
@@ -10,20 +10,23 @@
 # - https://code.google.com/p/jrfonseca/issues/detail?id=19
 # - http://docs.python.org/2/distutils/packageindex.html
 #
+# Original version was written by Jose Fonseca; see
+# https://github.com/jrfonseca/gprof2dot
 
 import sys
 from setuptools import setup
 
 setup(
-    name='gprof2dot',
-    version='2015.02.03',
-    author='Jose Fonseca',
-    author_email='jose.r.fonseca@gmail.com',
-    url='https://github.com/jrfonseca/gprof2dot',
-    description="Generate a dot graph from the output of several profilers.",
+    name='yelp-gprof2dot',
+    version='1.0.0',
+    author='Yelp Performance Team',
+    author_email='team-perf@yelp.com',
```

Comment:
I meant get rid of the "author_email" key entirely. I don't think team-perf is reachable from outside is it?